.github/workflows/run_xcodebuild_test_platforms.yml

```diff
@@ -16,9 +16,19 @@ on:
         required: false
         type: boolean
         default: false
+      retry_on_error:
+        description: 'Whether to automatically retry if the tests fail'
+        required: false
+        type: boolean
+        default: true
+      other_flags:
+        required: false
+        type: string
+        default: ''
 
 env:
   SCHEME: ${{ inputs.scheme }}
+  RETRY_ON_ERROR: ${{ inputs.retry_on_error }}
```

Comment:
GitHub is weird and would just not let me directly use `continue-on-error: ${{ inputs.retry_on_error }}` on the steps, so 🤷‍♂️